articles/azure-functions/functions-runtime-install.md

```diff
@@ -22,74 +22,74 @@ ms.author: anwestg
+The Azure Functions Runtime Preview Installer guides you through the installation of the Azure Functions Runtime preview Management and Worker Roles.  It is possible to install the Management and Worker role on the same machine.  However, as you add more Functions, you must deploy more worker roles on additional machines to be able to scale your functions onto multiple workers.
 
 ## Install the Management and Worker Role on the same machine
 
 1. Run the Azure Functions Runtime Preview Installer.
 
     ![Azure Functions Runtime Preview Installer][1]
 
-1. **Click Next** advance past the first stage of the installer
-1. Once you have read the terms of the **EULA**, **check the box** to accept the terms and **click Next** to advance.
-1. Now select the roles you want to install on this machine **Functions Management Role** and/or **Functions Worker Role** and **Click Next**
+1. Click **Next** advance past the first stage of the setup.
+1. Once you have read the terms of the **EULA**, **check the box** to accept the terms and click **Next** to advance.
+1. Now select the roles you want to install on this machine **Functions Management Role** and/or **Functions Worker Role** and click **Next**.
 
     ![Azure Functions Runtime Preview Installer - Role Selection][3]
 
     > [!NOTE]
-    > You can install the **Functions Worker Role** on many other machines to do so, follow these instructions, and only select **Functions Worker Role** in the installer.
+    > You can install the **Functions Worker Role** on many other machines. To do so, follow these instructions, and only select **Functions Worker Role** in the installer.
 
-1. **Click Next** to have the **Azure Functions Runtime Installer** install on your machine.
-1. Once complete the installer will launch the **Azure Functions Runtime Configuration tool**.
+1. Click **Next** to have the **Azure Functions Runtime Setup** begin the installation process on your machine.
+1. Once complete, the setup wizard will launch the **Azure Functions Runtime configuration tool**.
 
     ![Azure Functions Runtime Preview Installer Complete][5]
 
     > [!NOTE]
-    > If you are installing on **Windows 10** and the **Container** feature has not been previously enabled, the **Azure Functions Runtime** Installer prompts you to reboot your machine to complete the install.
+    > If you are installing on **Windows 10** and the **Container** feature has not been previously enabled, the **Azure Functions Runtime Setup** prompts you to reboot your machine to complete the install.
 
 ## Configure the Azure Functions Runtime
 
 To complete the Azure Functions Runtime installation you must complete the configuration.
 
-1. The **Azure Functions Runtime Configuration Tool** shows which roles are installed on your machine.
+1. The **Azure Functions Runtime configuration tool** shows which roles are installed on your machine.
```

Comment:
Given that "configuration tool" isn't in the UI, it might be better to exclude it from the bold formatting.